worker/gc/gc.go

```diff
@@ -90,3 +89,43 @@ func (g *GarbageCollector) DelPvPvcByServiceID(serviceGCReq model.ServiceGCTaskB
 		logrus.Warningf("service id: %s; delete a collection fo PVC: %v", serviceGCReq.ServiceID, err)
 	}
 }
+
+// DelKubernetesObjects deletes all kubernetes objects.
+func (g *GarbageCollector) DelKubernetesObjects(serviceGCReq model.ServiceGCTaskBody) {
```

Comment:
not delete `configmap` and other resources?

delete endpoint after delete service.

Reply:
Added configmaps, secrets and hpa, there should be nothing more